### PR TITLE
POWR-204 updates to the README.md to help onboard devs.

### PR DESCRIPTION
**Pull request by joshuami**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,90 +1,133 @@
...
-7. When you are done with your development for the day, run `Lando stop` to shut off your development containers.
+6. Add a settings.local.php
+    - Ask a teammate for a copy of a sane file to put at `/web/sites/default/settings.local.php`. Depending on whether you do a lot of theming you may want that file to turn on twig debugging and turn off several layers of caching.
+5. You have two options to get your database and files set up:
+    1. Lando quick start:
+        1. Run `lando pull` to get the DB and files from pantheon. This process takes a while. #grabsomecoffee
+    1. Manually import the database and files
+        1. Download the latest database and files backups from the Dev environment on Pantheon.
+        2. Move your database export into a folder in your project root called `/artifacts`. (We added this to our .gitignore, so the directory won't be there until you create it.)
+        3. From the project root, run `lando db-import portlandor_dev_2018-04-12T00-00-00_UTC_database.sql.gz`. (This is just an example, you'll need to use the actual filename of the database dump you downloaded.)
```
Comment (RichardDavies) [inline README.md]: From within the artifacts directory, run...

File: README.md [hunk 2]
```diff
@@ -1,90 +1,133 @@
...
+6. Add a settings.local.php
+    - Ask a teammate for a copy of a sane file to put at `/web/sites/default/settings.local.php`. Depending on whether you do a lot of theming you may want that file to turn on twig debugging and turn off several layers of caching.
+5. You have two options to get your database and files set up:
+    1. Lando quick start:
+        1. Run `lando pull` to get the DB and files from pantheon. This process takes a while. #grabsomecoffee
+    1. Manually import the database and files
+        1. Download the latest database and files backups from the Dev environment on Pantheon.
+        2. Move your database export into a folder in your project root called `/artifacts`. (We added this to our .gitignore, so the directory won't be there until you create it.)
+        3. From the project root, run `lando db-import portlandor_dev_2018-04-12T00-00-00_UTC_database.sql.gz`. (This is just an example, you'll need to use the actual filename of the database dump you downloaded.) 
+        4. Move your files backup into `web/sites/default/files`
```
Comment (RichardDavies) [inline README.md]: What is your "files backup"? How is this backup created and where is it located?

File: README.md (hunk 1 above)
Comment (joshuami) [reply README.md]: Whoops. I don't think you have to explicitly run it from within that director if you include the director in the path. I will update it to be `lando db-import artifacts/portlandor_dev_2018-04-12T00-00-00_UTC_database.sql.gz`, which should work for everyone.

Comment (RichardDavies) [reply README.md]: On Windows it doesn't work unless it's run from within the artifacts directory for some reason. Weird, I know.

Comment (RichardDavies) [reply README.md]: And the other mention of this command already says to run it from within artifacts so trying to remain consistent.

Comment (joshuami) [reply README.md]: Good to know. I've updated it to say from within artifacts directory and updated the command accordingly.

File: README.md (hunk 2 above)
Comment (RichardDavies) [reply README.md]: An extra "the" exists in "From the within the artifacts directory," But I'll leave it up to you whether or not to fix this minor grammatical error.

**Pull request merged**